backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHBackend.scala

```diff
@@ -129,7 +131,8 @@ object CHBackendSettings extends BackendSettingsApi with Logging {
       format: ReadFileFormat,
       fields: Array[StructField],
       partTable: Boolean,
-      paths: Seq[String]): Boolean = {
+      paths: Seq[String],
+      rootPaths: Seq[Path]): Boolean = {
```

Comment:
Minor: Do we have to have `paths` and  `rootPaths` together in the parameter list? Would it be reasonable to use the latter to extract the former?

Reply:
`paths` has a large initialization cost and is empty by default in velox backend.